src/types.jl

```diff
@@ -37,7 +37,7 @@ SCSMatrix(m::ManagedSCSMatrix) =
     SCSMatrix(pointer(m.values), pointer(m.rowval), pointer(m.colptr), m.m, m.n)
 
 
-struct SCSSettings
+mutable struct SCSSettings
     normalize::Int # boolean, heuristic data rescaling: 1
```

Comment:
Please update these comments to reflect that the default parameters are set by `SCS_set_default_settings!` (i.e., remove the values because we have no way to make sure they are correct).